src/chart/generateCategoricalChart.tsx

```diff
@@ -1094,9 +1088,9 @@ export const generateCategoricalChart = ({
   return class CategoricalChartWrapper extends Component<CategoricalChartProps, CategoricalChartState> {
     static displayName = chartName;
 
-    clipPathId: string;
+    readonly eventEmitterSymbol: Symbol = Symbol('rechartsEventEmitter');
```

Comment:
what does this do?

Reply:
I added this to prevent calling applySyncEvent when syscMethod is not a function.